<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>APIConfDialog</class>
 <widget class="QDialog" name="APIConfDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCheckBox" name="api_checkBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Push Tag Change to API</string>
   </property>
  </widget>
  <widget class="QComboBox" name="network_interface_Combobox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>401</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Network Interface</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>URL</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="api_url_lineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>401</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Data Format</string>
   </property>
  </widget>
  <widget class="QComboBox" name="data_format_Combobox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>401</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="update_pushButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>210</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>UPDATE</string>
   </property>
  </widget>
  <widget class="QLabel" name="update_message_level">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
